web/templates.go

```diff
@@ -264,13 +264,17 @@ document.onkeydown=function(e){
     <div class="container">
       {{template "footerBoilerplate"}}
       <p class="navbar-text navbar-right">
-      Took {{.Stats.Duration}}{{if .Stats.Wait}}(queued: {{.Stats.Wait}}){{end}} for
+      Took {{.Stats.Duration}}{{if .Stats.Wait}} (queued: {{.Stats.Wait}}){{end}} for
```

Comment:
is this space intentional

Reply:
yes, there is another parens a couple of lines below where we leave an extra space, too. I thought this looked nicer.